aiplatform/api/AIPlatform.Samples/PredictChatPromptSample.cs

```diff
@@ -0,0 +1,151 @@
+
+        // Or, you can construct Protobuf directly
+        var instance = new wkt::Value
+        {
+            StructValue = new()
+            {
+                Fields =
+                {
+                    ["context"] = wkt::Value.ForString("My name is Miles. You are an astronomer, knowledgeable about the solar system."),
+                    ["examples"] = wkt::Value.ForList(wkt::Value.ForStruct(new()
+                    {
+                        Fields =
+                        {
+                            ["input"] = wkt::Value.ForStruct(new()
+                            {
+                                Fields =
+                                {
+                                    ["content"] = wkt::Value.ForString("How many moons does Mars have?")
+                                }
+                            }),
+                            ["output"] = wkt::Value.ForStruct(new()
+                            {
+                                Fields =
+                                {
+                                    ["content"] = wkt::Value.ForString("The planet Mars has two moons, Phobos and Deimos.")
+                                }
+                            })
+                        }
+                    })),
+                    ["messages"] = wkt::Value.ForList(wkt::Value.ForStruct(new()
+                    {
+                        Fields =
+                        {
+                            ["author"] = wkt::Value.ForString("user"),
+                            ["content"] = wkt::Value.ForString(prompt),
+                        }
+                    }))
+                }
+            }
+        };
+
+        var instances = new List<wkt::Value>
+        {
+            instance
+        };
+
+        // You can construct Protobuf from JSON
+        var parametersJson = JsonConvert.SerializeObject(new
+        {
+            temperature = 0.3,
+            maxDecodeSteps = 200,
+            topP = 0.8,
+            topK = 40
+        });
+        // var parameters = wkt::Value.Parser.ParseJson(parametersJson);
+
+        // Or, you can construct Protobuf directly
+        var parameters = new wkt::Value
+        {
+            StructValue = new wkt::Struct
+            {
+                Fields =
+                {
+                    { "temperature", new wkt::Value { NumberValue = 0.3 } },
+                    { "maxDecodeSteps", new wkt::Value { NumberValue = 200 } },
+                    { "topP", new wkt::Value { NumberValue = 0.8 } },
+                    { "topK", new wkt::Value { NumberValue = 40 } }
+                }
+            }
+        };
+
+        // Make the request
+        var response = client.Predict(endpoint, instances, parameters);
+
+        // Parse and return the response
+        var content = response.Predictions[0].StructValue.Fields["candidates"].ListValue.Values[0].StructValue.Fields["content"].StringValue;
```

Comment:
Ah! This is hard.

Reply:
I tried to match the Java version of the samples that return the response..I'm parsing the content, to show people how to do it, as it's not obvious at all.

Reply:
Actually, Java version doesn't return anything..so I agree with you. We should just parse what we need and return that instead. That's what I'm doing now.